src/assets/scss/layout/_layout.scss

```diff
@@ -8,11 +8,12 @@
 // ---------------------------------------------------------
 .header {
   .pf-c-page__header-brand-toggle .pf-c-button {
-    margin-right: 0;
-    @media screen and (min-width: 992px) {
-      display: none !important;
+      margin-right: 0;
+      @media screen and (min-width: 992px) {
+        display: none !important;
+      }
     }
-  }
+  display: flex !important;
```

Comment:
Whenever there are nested selectors, put nested selectors at the end. Ensure to leave a whitespace between your rule declaration and nested selectors.